regrowl/bridge/netcent.py

```diff
@@ -0,0 +1,74 @@
+"""
+Echo a growl to OSX NotificationCenter
+Created by heilerich (https://github.com/heilerich)
+Requires PyObjC, install with pip
+
+Config Example:
+[regrowl.bridge.netcent]
+verbose = True
+"""
+
+from __future__ import absolute_import
+
+import logging
+
+try:
+    import Foundation
+    import objc
+    import AppKit
+except ImportError:
+    raise ImportError('Requires PyObjC, install with pip')
+
+from regrowl.regrowler import ReGrowler
+
+logger = logging.getLogger(__name__)
+
+__all__ = ['NetCenter']
+
+SPACER = '=' * 80
+
+
+class NetCenter(ReGrowler):
+    valid = ['REGISTER', 'NOTIFY']
+
+    def instance(self, packet):
+        pass
+
+    def register(self, packet):
+        logger.info('Register')
+        print 'Registration Packet:'
```

Comment:
I imagine most of these print statements should probably be switched to regular logger. statements? I think only the test "echo" bridge should probably use print.